app/models/order.rb

```diff
@@ -1,7 +1,7 @@
 class Order < ActiveRecord::Base
   has_many   :orders_items
   has_many   :purchases
-  has_many   :items, through: :orders_items
+  has_many   :items, through: :purchases
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.